### ISSUE TITLE
Check if PythonExporter from nbconvert can be used instead of our custom reimplementation

### ISSUE THREAD

**Issue by plutasnyy** [Adamage/tutorials#16]

**Comment by plutasnyy** [comment]
In my opinion, it's not worth to do that. The only way to simplify the code is to use `PythonExporter` with appropriate preprocessors to remove the markdown cells. Unfortunately, `TagRemovePreprocessor.remove_cell_tags` only looks at `tags`, and the cell type is set in the `cell_type` field. On the other hand `RegexRemovePreprocessor` according to my understanding  analyzes the `source` field. 
In summary, I would stay with our implementation.
